cmd/influxd/main.go

```diff
@@ -61,7 +62,7 @@ type Main struct {
 func NewMain() *Main {
 	return &Main{
 		Logger: zap.New(
-			zap.NewTextEncoder(),
+			zaplogfmt.NewEncoder(),
```

Comment:
Nope. But I'll modify the one in `cmd/influxd/run/server.go` anyway. The rest are in test files.